pyproject.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT"
 readme = "README.md"
```

Comment:
@proy9714 can you also please update requirements.txt file?

Reply:
Ok will do